stdlib/public/core/Span/Properties/InlineTypeExtensions.swift

```diff
@@ -0,0 +1,123 @@
+//===--- InlineTypeExtensions.swift ---------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+extension CollectionOfOne {
+
+  @available(SwiftStdlib 6.2, *)
+  public var storage: Span<Element> {
+    @_alwaysEmitIntoClient
+    @lifetime(borrow self)
+    borrowing get {
+      let pointer = UnsafePointer<Element>(Builtin.addressOfBorrow(self))
+      let span = Span(_unsafeStart: pointer, count: 1)
+      return _overrideLifetime(span, borrowing: self)
+    }
+  }
+}
+
+extension SIMD2 {
+
+  @available(SwiftStdlib 6.2, *)
+  public var storage: Span<Scalar> {
+    @_alwaysEmitIntoClient
+    @lifetime(borrow self)
+    borrowing get {
+      let pointer = UnsafePointer<Scalar>(Builtin.addressOfBorrow(self))
+      let span = Span(_unsafeStart: pointer, count: 2)
+      return _overrideLifetime(span, borrowing: self)
+    }
+  }
+}
```

Comment:
The generic SIMD types don't require *inline* or contiguous storage. For example, [SIMD-nontrivial.swift](https://github.com/swiftlang/swift/blob/swift-6.0.3-RELEASE/test/stdlib/SIMD-nontrivial.swift) uses an `Array<Scalar>`. Could these properties be moved into the existing [SIMDConcreteOperations.swift.gyb](https://github.com/swiftlang/swift/blob/swift-6.0.3-RELEASE/stdlib/public/core/SIMDConcreteOperations.swift.gyb) extensions? (Where the storage is known to be either `Builtin.Vec…xInt…` or `Builtin.Vec…xFPIEEE…`)

Reply:
Sorry for missing this earlier. You are correct that these would have to be defined specifically on the concrete types; it isn't clear whether that is worth the code size, so we will defer the SIMD-specific extensions until later. We may define them on the concrete type, or we may fix the `SIMD` protocol, or we may take advantage of `InlineArray` instead.

Reply:
> … we will defer the SIMD-specific extensions until later.

Another issue was that your extensions in this file would have needed either:

* [`#if SWIFT_STDLIB_ENABLE_VECTOR_TYPES`](https://github.com/swiftlang/swift/blob/swift-6.0.3-RELEASE/stdlib/public/core/StringUTF16View.swift#L725) in Swift, or

* [`if(SWIFT_STDLIB_ENABLE_VECTOR_TYPES)`](https://github.com/swiftlang/swift/blob/swift-6.0.3-RELEASE/stdlib/public/core/CMakeLists.txt#L257) in CMake.